tests/component-tests/pager/js/PagedTableTests.js

```diff
@@ -150,59 +159,101 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
     };
 
     fluid.tests.tooltipModuleSource = function (pager) {
-        var pageLinksTop = $("a", pager.pagerBar.locate("pageLinks"));
-        var pageLinksBottom = $("a", pager["pagerBar-1"].locate("pageLinks"));
-
-        var tooltipContents = [
-            [
-                {nodeName: "b", nodeText: "bird"},
-                {nodeName: "b", nodeText: "fish"}
+        
+        var expectedTooltipContents = {
+            size2: [
+                [ {nodeName: "b", nodeText: "dog"}, {nodeName: "b", nodeText: "cat"} ],
+                [ {nodeName: "b", nodeText: "bird"}, {nodeName: "b", nodeText: "fish"} ],
+                [ {nodeName: "b", nodeText: "camel"}, {nodeName: "b", nodeText: "dragon"} ],
+                [ {nodeName: "b", nodeText: "ant"}, {nodeName: "b", nodeText: "ant"} ]
+            ],
+            size3: [
+                [ {nodeName: "b", nodeText: "dog"}, {nodeName: "b", nodeText: "bird"} ],
+                [ {nodeName: "b", nodeText: "fish"}, {nodeName: "b", nodeText: "dragon"} ],
+                [ {nodeName: "b", nodeText: "ant"}, {nodeName: "b", nodeText: "ant"} ]
+            ],
+            size4: [
+                [ {nodeName: "b", nodeText: "dog"}, {nodeName: "b", nodeText: "fish"} ],
+                [ {nodeName: "b", nodeText: "camel"}, {nodeName: "b", nodeText: "ant"} ]
             ]
-        ];
+        };
 
         var sequence = [];
 
-        var assertVisibleTips = function (message, targetIds) {
+        var assertVisibleTips = function (message, targetIds, expectedTooltipContents, index) {
             sequence.push({
                 event: "{trackTooltips}.events.notifyFocusChange",
                 listener: function () {
                     fluid.tests.tooltip.assertVisible(message, pager, targetIds, null, function (tooltip) {
-                        jqUnit.assertNode("The contents of the tooltip should be set", tooltipContents[0], $("b", tooltip));
+                        jqUnit.assertNode("When the page size is " + pager.model.pageSize + ", The contents of the tooltip should be set", expectedTooltipContents[index], $("b", tooltip));
```

Comment:
Running the test shows output test messages like "Expected number of nodes When the page size is 2, The contents of the tooltip should be set", which does not make much sense.

This comment also applies to other output messages.